src/tests/ftest/launch.py

```diff
@@ -911,8 +911,39 @@ def archive_logs(avocado_logs_dir, test_yaml, args):
 
     # Copy any log files written to the DAOS_TEST_LOG_DIR directory
     logs_dir = os.environ.get("DAOS_TEST_LOG_DIR", DEFAULT_DAOS_TEST_LOG_DIR)
+
     archive_files(destination, host_list, "{}/*.log*".format(logs_dir))
 
+    # Caution: the glob expression "_output.log" must match the
+    #   --output-filename specified in # cart_utils.py:get_env()
+    tar_files(destination, "{}/*_output.log*".format(logs_dir))
+
+def tar_files(destination, logs_glob_expr):
+    """Run tar on files (in the directory specified by the orterun
+        --output-filename option) to the avocado results directory.  Useful for
+        copying, e.g., files from orterun --output-filename option to the avocado
+        results directory.
+
+    Args:
+        destination (str): (glob expression) path of files to be tar'd
+        logs_dir (str): destination of tar file(s)
+    """
+
+    # Create the destination directory
+    if not os.path.exists(destination):
+      get_output(["mkdir", destination])
+
+    import glob
+
+    for f_glob in glob.glob(logs_glob_expr):
```

Comment:
Isn't `f_glob` going to be each of the items the _glob_ matches?  So, are you creating a single file tar file for each log that matches the glob here?

Reply:
> Isn't `f_glob` going to be each of the items the _glob_ matches? So, are you creating a single file tar file for each log that matches the glob here?

The `*.log` files are actually directories, so each tar file will contain at least two files.  `orterun` writes stdout/stderr for each rank, e.g.,

```
/var/tmp/daos_testing/iv_basic_sep_output.log/1
/var/tmp/daos_testing/iv_basic_sep_output.log/1/rank.0
/var/tmp/daos_testing/iv_basic_sep_output.log/1/rank.0/stderr
/var/tmp/daos_testing/iv_basic_sep_output.log/1/rank.0/stdout
/var/tmp/daos_testing/iv_basic_sep_output.log/1/rank.1
/var/tmp/daos_testing/iv_basic_sep_output.log/1/rank.1/stderr
/var/tmp/daos_testing/iv_basic_sep_output.log/1/rank.1/stdout
...
```

So we want an archive of each `*_output.log` directory.

Reply:
I see.  Fair enough.